{{cookiecutter.project_slug}}/.{{ cookiecutter.project_slug }}_dev.env

```diff
@@ -0,0 +1,3 @@
+export API_KEY="ApiKey XXXXXXX:xxxxxxxxxxxxxxxxxxxxxxxxxxx"
```

Comment:
# In order to use this extension locally, you must obtain an API Key from the integration module of your account. Please use a Custom token when running locally.
export API_KEY="ApiKey XXXXXXX:xxxxxxxxxxxxxxxxxxxxxxxxxxx"

# You can obtain your environment ID going to DevOps module, there you will find this key for the concrete environment you want to use, either development, testing, or production

export ENVIRONMENT_ID=xxxxxxx-xxxx-xxxx-xxxx

# Please provide the hostname of Connect API. This one can be found in the general section on the integrations module
export SERVER_ADDRESS=api.connect.cloudblue.com

# Please note that this file shall ONLY be present when running locally and never in a cloud environment.